packages/openapi-ts/src/getSpec.ts

```diff
@@ -57,6 +57,7 @@ export const getSpec = async ({
         };
       }
       response = request.response;
+      console.log('Request HEAD response', response);
```

Comment:
I assume you want to remove this?

Reply:
That's what you get for committing before your morning coffee...

Yeah I'll push a fix, thanks